blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderRunContainerImpl.java

```diff
@@ -33,7 +38,17 @@ public Link getLink() {
 
     @Override
     public BlueRun create(StaplerRequest request) {
-        return null;
+        pipeline.folder.checkPermission(Item.BUILD);
+        Queue.Item queueItem = pipeline.folder.scheduleBuild2(0, new CauseAction(new Cause.UserIdCause()));
+        if(queueItem == null){ // possible folder.isBuildable() was false due to no repo fetched yet
+            return null;
+        }
+        return new QueueItemImpl(
+                pipeline.getOrganization(),
+                queueItem,
+                pipeline,
+                1
+        ).toRun();
```

Comment:
Users who have pre-existing blueocean created org folder pipeline, we want them to be able to be able to edit and trigger indexing. This change triggers org folder indexing.